features/admin.claims.v1/components/wizard/local-claim/basic-details-local-claims.tsx

```diff
@@ -165,14 +165,27 @@ export const BasicDetailsLocalClaims = (props: BasicDetailsLocalClaimsPropsInter
             onSubmit={ (values: Map<string, FormValue>) => {
                 const data: Claim
                  = {
+                     canonicalValues: values.get("canonicalValues")
+                         ? ((values.get("canonicalValues") as unknown) as { key: string; value: string }[])
+                             .map((item: { key: string; value: string }) => ({
+                                 key: item.key,
+                                 value: item.value
+                             }))
+                         : [],
                      claimURI: claimURIBase + "/" + values.get("claimURI").toString().trim(),
+                     dataType: values.get("dataType")?.toString() === DataType.STRING
+                         ? (values.get("canonicalValues") && (values.get("canonicalValues") as string[]).length > 0
```

Comment:
Will we be able to simplify as follows?

```suggestion
                         ? (values.get("canonicalValues") as string[]).length > 0
```

Reply:
Fixed by - https://github.com/wso2/identity-apps/pull/8111/commits/6f8d1b53919c0d00e80b9142da52b064cef38ebf